src/api/Gemfile.lock

```diff
@@ -116,11 +116,11 @@ GEM
     faker (1.7.3)
       i18n (~> 0.5)
     feature (1.4.0)
+    ffi (1.9.17)
```

Comment:
I have put the three bugfixes in https://github.com/openSUSE/open-build-service/pull/3031 now